README.md

```diff
@@ -193,3 +193,61 @@ $ telnet 192.168.99.100 1234
 
 This `-p` argument links port 4321 on the container to port 1234 on the VM network adapter. It must appear after `docker run` and before `dmscid/plankton`. This allows us to connect to the container from outside of the VM, in this case using Telnet. The `192.168.99.100` IP is the IP of the VM on the bridge network between the host and the VM. VirtualBox will typically use this IP when available, but it may be different on your system.
 
+## Remote access to devices
+
+*Please note that this functionality should only be used on a trusted network.*
+
+Besides the device specific protocols, the device can be made accessible from the outside via JSON-RPC over ZMQ. This can be achieved by passing the `-r` option with a `host:port` string to the simulation:
+
+```
+$ python simulation.py -r 127.0.0.1:10000 -d chopper -- -p SIM:
+```
+
+Now the device can be controlled via the `control.py`-script in a different terminal window. The service can be queried to show the available objects using the `-l` or `--list-objects` flag:
+
+```
+$ python control.py --ip 127.0.0.1 --port 10000 --list-objects
```

Comment:
Maybe you're right, although I would have liked to us I/p on the server side, but p is occupied. So for being consistent we should use r on the client.